client/src/components/Cropper/Cropper.tsx

```diff
@@ -2,8 +2,10 @@ import React from 'react';
 import Avatar from 'react-avatar-edit';
 import noAvatar from '../assets/noAvatar.svg';
 import {Button} from 'reactstrap';
-
-class Cropper extends React.Component {
+interface IFormProps {
```

Comment:
do we have form?

Reply:
Fiexed.

Reply:
https://github.com/jsgram/JSGram/pull/75